templates/pages/setup/webhook/webhook.html.twig

```diff
@@ -134,7 +134,9 @@
         field_options
     ) }}
 
-    {{ fields.dropdownYesNo('save_response_body', item.fields['save_response_body'], __('Save response body')) }}
+    {% if constant('GLPI_WEBHOOK_ALLOW_RESPONSE_SAVING') %}
+        {{ fields.dropdownYesNo('save_response_body', item.fields['save_response_body'], __('Save response body')) }}
+    {% endif %}
```

Comment:
If the feature is disabled, maybe the field should be displayed in a disabled state with a "no" forced value.

It would probably be easier to document this. Indeed, people that would not see this field may consider it is a bug ans ask for support, but if they see a disabled field and then see in documentation that it means that the feature is disabled at the system level, then they will be able to understand that it is not a bug.

Reply:
The ability to save responses is disabled by default. It should be documented, if at all, as a feature that is disabled by default and heavily discouraged to enable.